### DISCUSSION TITLE
CUYUNIVERSITY Semester 4 - SESI2 install

### DISCUSSION THREAD

**Discussion by Destaanugra1** [deaafrizal/forum#46] [Troubleshooting Room]
![Screenshot (127)](https://github.com/deaaprizal/forum/assets/152362800/633cf539-c6fc-445e-8f5c-9dfb216e81db)
kenapa ya bang saya sudah buat folder npm masih error

**Comment by andikaalakate** [comment]
@phosphor-icons/react seharusnya, kamu kurang s nya

**Reply by deaafrizal** [reply]
thank u @andikaalakate sudah bantu ✨
